pkg/webhook/server/certificate/manager.go

```diff
@@ -117,7 +117,7 @@ func (m *Manager) Start(stopCh <-chan struct{}) error {
 // of generating new one
 func (m *Manager) loadCACertFromCABundle() {
 	caBundle, err := m.CABundle()
-	if err != nil || len(caBundle) == 0 {
```

Comment:
This is not related and also `ParseCertsPEM` will fail if caBundle is empty,

Reply:
If I remove this line entirely then I think we could have an issue in ParseCertsPEM(), because there we ask `for len(caBundle) > 0 {` so we might fail on that